pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -240,6 +242,40 @@ func (b *OFBridge) DeleteMeterAll() error {
 	return nil
 }
 
+func (b *OFBridge) GetMeterStats(handleMeterStatsReply func(packetCounts map[int]int64)) error {
+	const OFPM_ALL = 0xffffffff // Represents all meters
+	mpMeterStatsReq := openflow15.NewMpRequest(openflow15.MultipartType_MeterStats)
+	meterMPReq := openflow15.NewMeterMultipartRequest(OFPM_ALL)
+	mpMeterStatsReq.Body = append(mpMeterStatsReq.Body, meterMPReq)
+	ch := make(chan *openflow15.MultipartReply, 1)
+	b.mpReplyChsMutex.Lock()
+	defer b.mpReplyChsMutex.Unlock()
+	b.multipartReplyChs[mpMeterStatsReq.Xid] = ch
```

Comment:
even though the cost of creating a goroutine is small, it's probably better not to hold the mutex while we do it:
```go
func() {
	b.mpReplyChsMutex.Lock()
	defer b.mpReplyChsMutex.Unlock()
	b.multipartReplyChs[mpMeterStatsReq.Xid] = ch
}()
```